.rubocop.yml

```diff
@@ -1,22 +1,3 @@
-AbcSize:
```

Comment:
We've completely removed the `.rubocop.yml` file in other projects that have been moved to ChefStyle. Wouldn't that also make sense here?